.travis.yml

```diff
@@ -66,7 +68,7 @@ before_install:
   - sudo apt-get update
   - sudo bash mocha_test/scripts/install_geth_$GETH_VERSION.sh
   - geth version
-  - sudo apt-get install solc
+  - sudo apt-get install solc -y
```

Comment:
If I understand correctly, the `-y` flag is to say yes automatically to prompts. Is that wise? Also, why is there a problem now (of "unauthenticated sources", if I read the commit comment correctly) but neither (a) before nor (b) for the other repos (i.e., ost-price-oracle)?